addons/udes_sale_stock/models/sale_order.py

```diff
@@ -334,10 +345,32 @@ def confirm_orders(self):
             # allow serialisation error to propagate.
             self.env.cr.commit()
             self.invalidate_cache()
+        if exception_data:
+            # Raise an exception including details of the exceptions that have
+            # suppressed, in case we want to expose this information somewhere.
```

Comment:
Is this comment missing a word?

Reply:
How about: 

> Raise an exception that includes details of the exceptions that have being suppressed, in case we want to expose this information somewhere else

?

Reply:
Sounds good. I thought a `been` had gotten lost in the original comment

Reply:
Hah, good point.   I can never see my own mistakes.

Although in the above, s/being/been :smile: